lib/external/math.dart

```diff
@@ -7,6 +7,16 @@ import 'package:Birb/utils/exceptions.dart';
 
 void registerMath(Runtime runtime) {
   registerGlobalFunction(runtime, 'rand', funcRand);
+  registerGlobalFunction(runtime, 'sin', funcSin);
+  registerGlobalFunction(runtime, 'cos', funcCos);
+  registerGlobalFunction(runtime, 'tan', funcTan);
+  registerGlobalFunction(runtime, 'asin', funcASin);
+  registerGlobalFunction(runtime, 'acos', funcACos);
+  registerGlobalFunction(runtime, 'atan', funcATan);
+  registerGlobalFunction(runtime, 'atan2', funcATan2);
+  registerGlobalFunction(runtime, 'exp', funcExp);
+  registerGlobalFunction(runtime, 'log', funcLog);
+  registerGlobalFunction(runtime, 'ln', funcLn);
```

Comment:
Please order these alphebetically

Reply:
Didn't noticed this but I've sorted them